### PR TITLE
Telegraf v1.31.0

### PR DESCRIPTION
**Pull request by powersj**
New minor release - thank you!

### CODE REVIEW

File: library/telegraf [hunk 1]
```diff
@@ -2,29 +2,26 @@ Maintainers: Josh Powers <jpowers@influxdata.com> (@powersj),
...
-Tags: 1.28-alpine, 1.28.5-alpine
-Architectures: amd64, arm64v8
-Directory: telegraf/1.28/alpine
-
 Tags: 1.29, 1.29.5
 Architectures: amd64, arm32v7, arm64v8
 Directory: telegraf/1.29
 
 Tags: 1.29-alpine, 1.29.5-alpine
-Architectures: amd64, arm64v8
```
Comment (tianon) [inline library/telegraf:21]: Just to be sure, this is intentionally dropped, right? (this will remove arm64v8 from 1.29)

Comment (yosifkit) [inline library/telegraf:21]: Is the removal of `arm64v8` here and on `1.30-alpine` intentional?

Comment (powersj) [reply library/telegraf:21]: definitely not intentional, pushed a change to re-add

Comment (powersj) [reply library/telegraf:21]: definitely not intentional, pushed a change to re-add

**Review by tianon — APPROVED** [review]

**Comment by denschub** [comment]
I see that the tag has been pushed 5 hours ago to https://hub.docker.com/layers/library/telegraf/1.31/images/sha256-eb37f58646a901dc7727cf448cae36daaefaba79de33b5058dab79aa4c04aefb?context=explore, but I see no layers in there, and I also can't pull the image - I just get a "no matching manifest for linux/amd64 in the manifest list entries".

Is there something wrong, or is this just some heavily delayed cache?

**Pull request merged**